tools/code-analyzer/src/lib/hook-changes.ts

```diff
@@ -101,14 +104,15 @@ export const scanForHookChanges = async (
 						}
 					} );
 
-					changes.set( filePath, {
+					changes.set( filePath + name, {
 						filePath,
 						name,
 						hookType,
 						description,
 						changeType,
 						version,
 						ghLink,
+						isPlaceholder,
```

Comment:
On Step 22 I get an error here when running `pnpm run analyzer scan hooks "test/hook-version" "trunk" --since "8.0.0"`.

```
ReferenceError: isPlaceholder is not defined
    at scanForHookChanges (/Users/paulsealock/Workspace/psealock-woocommerce/tools/code-analyzer/src/lib/hook-changes.ts:115:7)
    at async scanChangesForHooks (/Users/paulsealock/Workspace/psealock-woocommerce/tools/code-analyzer/src/lib/scan-changes.ts:87:22)
    at async Command.<anonymous> (/Users/paulsealock/Workspace/psealock-woocommerce/tools/code-analyzer/src/commands/analyzer/analyzer-scan.ts:83:26)
```